presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -440,6 +440,49 @@ public void testSplitToMap()
         assertCachedInstanceHasBoundedRetainedSize("SPLIT_TO_MAP('a=123,b=.4,c=,=d', ',', '=')");
     }
 
+    @Test
+    public void testSplitToMultimap()
+    {
+        MapType expectedType = mapType(VARCHAR, new ArrayType(VARCHAR));
+
+        assertFunction("SPLIT_TO_MULTIMAP('', ',', '=')", expectedType, ImmutableMap.of());
+        assertFunction(
+                "SPLIT_TO_MULTIMAP('a=123,b=.4,c=,=d', ',', '=')",
```

Comment:
Would it make sense to make test includes mixed single value and multiple value?  It looks like for other test cases that contains multiple values, there is only one key (line 458 and line 459) ?